lib/matplotlib/axis.py

```diff
@@ -1962,7 +1962,7 @@ def set_ticklabels(self, labels, *, minor=False, fontdict=None, **kwargs):
         labels : sequence of str or of `.Text`\s
             Texts for labeling each tick location in the sequence set by
             `.Axis.set_ticks`; the number of labels must match the number of
-            locations.
+            locations. The labels are used as is, without further formatting.
```

Comment:
Maybe make it explicit that a Formatter is being used, and hence if you use another it will obviate this one? 

```suggestion
            locations. The labels are used as is, and passed to a `.FixedFormatter`.  
```

Reply:
I went for a variant of @jklymak's wording.

Reply:
Looks like only one entry was changed?

Reply:
indeed, fixed.